src/bls.mjs

```diff
@@ -2500,54 +2381,38 @@ export async function handleSkaleImaBSU256( joCallData ) {
         } else
             optsBSU256.details.warning( "Will sign via SGX without SSL options" );
         const signerIndex = optsBSU256.imaState.nNodeNumber;
-        await rpcCall.create( optsBSU256.joAccount.strSgxURL, rpcCallOpts,
-            async function( joCall, err ) {
-                if( err ) {
-                    if( log.id != optsBSU256.details.id ) {
-                        log.error(
-                            "{p}JSON RPC call(handleSkaleImaBSU256) to SGX failed, " +
-                            "RPC call was not created, error is: {err}",
-                            optsBSU256.strLogPrefix, err );
-                    }
-                    optsBSU256.details.error(
-                        "{p}JSON RPC call(handleSkaleImaBSU256) to SGX failed, " +
-                        "RPC call was not created, error is: {err}",
-                        optsBSU256.strLogPrefix, err );
-                    if( joCall )
-                        await joCall.disconnect();
-                    throw new Error( "JSON RPC call(handleSkaleImaBSU256) to SGX failed, " +
-                        "RPC call was not created, " +
-                        `error is: ${owaspUtils.extractErrorMessage( err )}` );
-                }
-                const joCallSGX = {
-                    "jsonrpc": "2.0",
-                    "id": randomCallID(),
-                    "method": "blsSignMessageHash",
-                    "params": {
-                        "keyShareName": optsBSU256.joAccount.strBlsKeyName,
-                        "messageHash": optsBSU256.strMessageHash,
-                        "n": optsBSU256.nParticipants,
-                        "t": optsBSU256.nThreshold,
-                        "signerIndex": signerIndex + 0 // 1-based
-                    }
-                };
-                optsBSU256.details.trace( "{p}Will invoke SGX with call data {}",
-                    optsBSU256.strLogPrefix, joCallSGX );
-                await joCall.call( joCallSGX, async function( joIn, joOut, err ) {
-                    await handleBlsSignMessageHash256Result(
-                        optsBSU256, joCallData, joCall, joIn, joOut, err );
-                } ); // joCall.call ...
-            } ); // rpcCall.create ...
+        joCall = await rpcCall.create( optsBSU256.joAccount.strSgxURL, rpcCallOpts );
+        const joIn = {
+            "jsonrpc": "2.0",
+            "id": randomCallID(),
+            "method": "blsSignMessageHash",
+            "params": {
+                "keyShareName": optsBSU256.joAccount.strBlsKeyName,
+                "messageHash": optsBSU256.strMessageHash,
+                "n": optsBSU256.nParticipants,
+                "t": optsBSU256.nThreshold,
+                "signerIndex": signerIndex + 0 // 1-based
```

Comment:
Why adding `0` here?

Reply:
`const signerIndex = optsBSU256.imaState.nNodeNumber;` ... adding zero just guarantees no object reference
but here I will remove it

Reply:
removed